pygotham/api/sponsors.py

```diff
@@ -0,0 +1,32 @@
+"""Sponsor-related API endpoints."""
+
+from flask import Blueprint
+from flask_restful import Api, Resource
+
+from pygotham.events.models import Event
```

Comment:
Is there a reason not to use `from pygotham.models import Event, Level, Sponsor`?